yarf/rf_libraries/libraries/video_input_base.py

```diff
@@ -127,6 +127,36 @@ def _end_suite(self, data, result) -> None:
                 else:
                     self._log_video(video_path)
 
+    @keyword
+    def set_ocr_confidence_threshold(self, threshold: float) -> None:
+        """
+        Set the OCR confidence threshold.
+
+        Args:
+            threshold: Confidence threshold between 0 and 1.
+        """
+        logger.debug(f"Setting OCR confidence threshold to {threshold}")
+        self.ocr.DEFAULT_CONFIDENCE = threshold  # type: ignore[union-attr]
+
+    @keyword
+    def set_ocr_coincidence_threshold(self, threshold: float) -> None:
```

Comment:
What if we use a global variable instead? It won't help with tesseract being special, but we can mention it in the related documentation.